src/routes/api/user.routes.js

```diff
@@ -13,4 +15,16 @@ router.post(
 
 router.post('/login', getExistUser, checkUserVerified, checkValidPassword,UserController.loginUser);
 
+router.get(
+  '/', 
+  isAdmin,
```

Comment:
I am expecting two middlewares here to replace isAdmin
- protectRoute
- restrictedTo(<roles>)

Reply:
For protectRoute, we have auth.middleware , I think they may be performing same task